### PR TITLE
Add UNL CS Department

### PR DESCRIPTION
**Pull request by flaviomartins**
Added the entire UNL CS faculty

### CODE REVIEW

**Comment by fycus-tree** [comment]
A lot of these URLs are broken or you're missing scholar information. 
For example, the last person is Vítor Duarte. The given URL is https://asc.di.fct.unl.pt/~vad/ but that doesn't load for me. Google gives me https://docentes.fct.unl.pt/vdt, which does load. You also say there's no scholar page, but that URL says his scholar page is https://scholar.google.pt/citations?user=Wqer7_EAAAAJ

**Comment by flaviomartins** [comment]
@fycus-tree thanks for looking into this.

Unfortunately, it seems some UNL servers are down for maintenance. I've added the people and their pages as listed on https://www.di.fct.unl.pt/en/departamento/pessoas/docentes

There are two people named Vitor Duarte affiliated with the School of Science and Technology. I guess the one you found is with the Physics department. The one I'm listing is with CS.

File: csrankings.csv [hunk 1]
```diff
@@ -4043,6 +4051,7 @@ Fengyuan Ren,Tsinghua University,https://nns.cs.tsinghua.edu.cn/~renfy,NOSCHOLAR
 Ferda Nur Alpaslan,Middle East Technical University,http://user.ceng.metu.edu.tr/~alpaslan,NOSCHOLARPAGE
 Fereydoun Kazemian,Rochester Institute of Technology,https://www.cs.rit.edu/~fxk,NOSCHOLARPAGE
 Ferkan Yilmaz,Yıldız Technical University,https://www.ce.yildiz.edu.tr/personal/ferkan?lang=en,yD0td6wAAAAJ
+Fernanda Barbosa,Universidade NOVA de Lisboa
```
Comment (emeryberger) [inline csrankings.csv]: This entry is incomplete.

File: csrankings.csv [hunk 2]
```diff
@@ -6474,6 +6488,7 @@ Jordi Perello Muntan,Polytechnic University of Catalonia,http://people.ccaba.upc
 Jordi Torres,Polytechnic University of Catalonia,https://www.facebook.com/public/Jordi-Torres-Sanchez/school/Polytechnic-University-of-Catalonia-109077209117931,waODsw0AAAAJ
 Jorge Arturo Cobb,University of Texas at Dallas,http://www.utdallas.edu/~jcobb,NOSCHOLARPAGE
 Jorge C. Lucero,Universidade de Brasília,http://cic.unb.br/~lucero,YRGc9WwAAAAJ
+Jorge Cruz 0001,Universidade NOVA de Lisboa,https://userweb.fct.unl.pt/~jc,ENwGNokAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: It says "no permission" to access this URL. I find that I cannot access many of the listed home pages.

File: csrankings.csv [hunk 3]
```diff
@@ -6483,6 +6498,8 @@ Jorgen P. Bansler,University of Copenhagen,http://diku.dk/Ansatte/beskrivelse/?i
 Jorgen Peddersen,UNSW,https://www.engineering.unsw.edu.au/computer-science-engineering/staff/jorgen-peddersen,NOSCHOLARPAGE
 Joschka Boedecker,University of Freiburg,https://nr.informatik.uni-freiburg.de/people/joschka-boedecker,2mv2dDkAAAAJ
 Jose Dolz,ETS Montreal,https://www.etsmtl.ca/Professeurs/jdolz,yHQIFFMAAAAJ
+José Gabriel Pereira Lopes,Universidade NOVA de Lisboa,http://dev-hlt.di.fct.unl.pt/gpl/,LRk6X3QAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: This page brings up a directory; it's not a real home page.

**Review by emeryberger — COMMENTED** [review]
The current situation with the home pages is really unacceptable; I can't access most of them, and others are defective.

**Comment by emeryberger** [comment]
bump, @flaviomartins

**Comment by flaviomartins** [comment]
@emeryberger I have fixed all the home pages and rebased.

File: csrankings.csv [hunk 4]
```diff
@@ -6619,6 +6636,7 @@ José António Beltran Gerald,Universidade de Lisboa,http://ieeexplore.ieee.org/
 José Augusto Suruagy Monteiro,UFPE,http://www.di.ufpe.br/~suruagy,xCIQf-QAAAAJ
 José Bento,Boston College,http://www.jbento.net,8K6Z0lEAAAAJ
 José Bento Ayres Pereira,Boston College,http://www.jbento.net,8K6Z0lEAAAAJ
+José C. Cunha,Universidade NOVA de Lisboa,http://nova-lincs.di.fct.unl.pt/person/1,VeZ4QbAAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:6639]: Per his home page, he retired a number of years ago.
http://asc.di.fct.unl.pt/~jcc/

File: csrankings.csv [hunk 5]
```diff
@@ -6667,21 +6686,27 @@ Joydeep Biswas,University of Massachusetts Amherst,https://www.cics.umass.edu/pe
...
 João Henrique de Souza Pereira,UFU,http://www.portal.facom.ufu.br/node/420,NOSCHOLARPAGE
+João Leite,Universidade NOVA de Lisboa,http://nova-lincs.di.fct.unl.pt/person/174,ReF98QcAAAAJ
+João Lourenço,Universidade NOVA de Lisboa,http://nova-lincs.di.fct.unl.pt/person/17,8aN-HtMAAAAJ
 João Luiz Dihl Comba,UFRGS,http://www.inf.ufrgs.br/~comba,NOSCHOLARPAGE
 João Luís Garcia Rosa,USP-ICMC,http://conteudo.icmc.usp.br/pessoas/joaoluis,YsO1rOUAAAAJ
 João M. Lemos,Universidade de Lisboa,https://fenix.tecnico.ulisboa.pt/homepage/ist11886,NOSCHOLARPAGE
 João Madeiras Pereira,Universidade de Lisboa,http://lisboa.academia.edu/Jo%C3%A3oMadeirasPereira,kNDBE9kAAAAJ
 João Marcelo X. N. Teixeira,UFPE,http://www.cin.ufpe.br/voxarlabs,CQNizNEAAAAJ
 João Marcelo Xavier Natario Teixeira,UFPE,http://www.cin.ufpe.br/voxarlabs,CQNizNEAAAAJ
+João Magalhães,Universidade NOVA de Lisboa,http://nova-lincs.di.fct.unl.pt/person/46,P6fBAXMAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:6706]: Please replace these generic home pages with the actual personal home pages, e.g., http://ctp.di.fct.unl.pt/~jmag/

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Comment by emeryberger** [comment]
Closing for lack of response. Please feel free to reopen.

**Pull request closed without merging**